index.html

```diff
@@ -82,7 +82,7 @@ <h3>
                     <li>Ruby</li>
                 </ul>
                 </div>
-                <button>See Project</button>
+                <button id="portfolio-btn">See Project</button>
```

Comment:
- [ ] Well done so far on modifying your design as requested by the reviewer. However, please look at the image below and add the missing elements to match the design! There's a missing button please add it.
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/210605103-676a764d-667f-47a1-be6d-01f7869e2ebc.png)